apps/src/code-studio/pd/workshop_enrollment/workshop_details.jsx

```diff
@@ -82,6 +82,21 @@ export default class WorkshopDetails extends React.Component {
     );
   }
 
+  fee() {
+    if (this.props.workshop.course === 'CS Fundamentals') {
```

Comment:
It's so painful that we rely on comparisons to user-presented strings, rather than internal identifiers specified using constants.

Reply:
Maybe I'm not understanding, but couldn't you use constants we import from our Ruby code to JS, like [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx#L11)?

Reply:
Certainly, we should.

However, that file actually just seems to hardcode the same string for general use [here](https://github.com/code-dot-org/code-dot-org/blob/61f50d8fa14716bf13cac13cc686154a7b3fb4f0/apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx#L17).

We come closest with this constant in Ruby [here](https://github.com/code-dot-org/code-dot-org/blob/61f50d8fa14716bf13cac13cc686154a7b3fb4f0/lib/cdo/shared_constants/pd/shared_workshop_constants.rb#L4).  However, for some reason when we generate `sharedWorkshopConstants.js` from that, we get this:

```
export const Courses = [
  "CS Fundamentals",
  "CS Principles",
  "Exploring Computer Science",
  "CS in Algebra",
  "CS in Science",
  "CS Discoveries",
  "Counselor",
  "Admin",
  "Facilitator"
];
```
We can't rely on array index position to know which one is CSF, and as such, I don't see any way to get this string from a reliable source of truth.

And even if we could, it still wouldn't excuse using a human-readable string as the key, rather than a simpler key value.  Unfortunately, even this very file [already](https://github.com/code-dot-org/code-dot-org/blob/61f50d8fa14716bf13cac13cc686154a7b3fb4f0/apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx#L388) [did](https://github.com/code-dot-org/code-dot-org/blob/61f50d8fa14716bf13cac13cc686154a7b3fb4f0/apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx#L446) that, so this PR is just following precedent.